### PR TITLE
Adding google play button to readme

### LINKED ISSUE #4: Add "google play" button in read me
https://play.google.com/store/apps/details?id=com.shreyasvisweshwaran.matchdapairs

### PR DESCRIPTION
**Pull request by BIJJUDAMA**
@FirefoxSRV Completed #4 
Can you please review

### CODE REVIEW

File: README.md
Comment (FirefoxSRV) [inline README.md:1]: Instead of text link, add a "Get it on Google play" button

**Comment by FirefoxSRV** [comment]
https://partnermarketinghub.withgoogle.com/brands/google-play/visual-identity/badge-guidelines/

File: README.md
Comment (BIJJUDAMA) [reply README.md:1]: Ohh ok understood

**Comment by BIJJUDAMA** [comment]
@FirefoxSRV Can you check now please?

**Comment by FirefoxSRV** [comment]
The play button is too big.....an appropriate size is affirmative

**Comment by BIJJUDAMA** [comment]
@FirefoxSRV Is this fine? Please let me know
Thanks!!

**Review by FirefoxSRV — APPROVED** [review]
Looks good !

**Pull request merged**